test/suite/test_error_info.py

```diff
@@ -34,13 +34,95 @@
 # test_error_info.py
 #   Test that the placeholder get_last_error() session API returns placeholder error values.
 class test_error_info(compact_util):
-    table_name1 = 'table:test_error'
 
-    def check_error(self, error, sub_level_error, error_msg):
+    table_name1 = 'table:test_error_infoa.wt'
+    table_name2 = 'table:test_error_infob.wt'
+    table_name3 = 'table:test_error_infoc.wt'
+
+    repeat_call = False
+    ERROR_INFO_EMPTY = ""
+    ERROR_INFO_SUCCESS = "last API call was successful"
+    EINVAL_message = "unknown configuration key 'expect_this_error'"
+    EBUSY_message = "the table has uncommitted data and cannot be dropped yet"
+
+    def assert_error_equal(self, err_val, sub_level_err_val, err_msg_val):
         err, sub_level_err, err_msg = self.session.get_last_error()
-        self.assertEqual(err, error)
-        self.assertEqual(sub_level_err, sub_level_error)
-        self.assertEqual(err_msg, error_msg)
+        self.assertEqual(err, err_val)
+        self.assertEqual(sub_level_err, sub_level_err_val)
+        self.assertEqual(err_msg, err_msg_val)
+
+    def cursor_s(self, tablename, key):
+        cursor = self.session.open_cursor(tablename, None, None)
+        cursor.set_key(key)
+        return cursor
+
+    def api_call_with_success(self):
+        """
+        Create a table, add a key, get it back
+        """
+        self.session.create(self.table_name1, 'key_format=S,value_format=S')
+        inscursor = self.cursor_s(self.table_name1, 'key1')
+        inscursor.set_value('value1')
+        inscursor.insert()
+        inscursor.close()
+        getcursor = self.cursor_s(self.table_name1, 'key1')
+        getcursor.search()
+        getcursor.close()
+
+    def api_call_with_EINVAL(self):
+        expectMessage = 'unknown configuration key'
+        with self.expectedStderrPattern(expectMessage):
+            try:
+                self.session.create(self.table_name2, 'expect_this_error,okay?')
+            except wiredtiger.WiredTigerError as e:
+                self.assertTrue(str(e).find('nvalid argument') >= 0)
```

Comment:
Did you miss the first letter on purpose to ignore the case?

```suggestion
                self.assertTrue(str(e).find('nvalid argument') >= 0)
```

Reply:
Honestly I copied that snippet from another test so I don't know why the first letter is missing.
I [added it back in](https://github.com/wiredtiger/wiredtiger/pull/11406/commits/5f9dc94e2684dcf4b88c2d7fe09b8a48d9888aa8) and the test still passed, so probably just a typo.